src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -250,12 +269,13 @@ public void validate() throws AppEngineException {
     private int runDevAppServerWaitSeconds;
 
     /**
-     * The home directory of Google Cloud SDK. If not set, will attempt to look for the SDK in known
-     * install locations.
+     * The home directory of Google Cloud SDK.
+     *
+     * @param sdkPath the root path for the Cloud SDK
      */
-    public Builder sdkPath(File sdkPathFile) {
-      if (sdkPathFile != null) {
-        this.sdkPath = sdkPathFile.toPath();
+    public Builder sdkPath(Path sdkPath) {
```

Comment:
The Api to the library was accepting File instead of Path. Having this builder accept path affects that consistency. Having said that, it's not a really big deal. I would have liked this CL to be more constrained in it's purpose though. It really seems to have pushed a lot of things in that could have been discussed.